composer.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "bshaffer/oauth2-server-bundle",
+    "name": "alpha1125/oauth2-server-bundle",
```

Comment:
you are right, it should not be expected.  That should not be in there... "bshaffer" would be the correct one.  Accidentally committed back in from testing... how do I revert that.